services/graph/pkg/identity/ldap_group.go

```diff
@@ -423,7 +423,7 @@ func (i *LDAP) getGroupsForUser(dn string) ([]*ldap.Entry, error) {
 		"(%s=%s)",
 		i.groupAttributeMap.member, dn,
 	)
-	userGroups, err := i.getLDAPGroupsByFilter(groupFilter, false, false)
+	userGroups, err := i.getLDAPGroupsByFilter(groupFilter, true, false)
```

Comment:
With the above suggestion for the Modify request, you could change this back to `false` again.